test/python/primitives/test_statevector_sampler.py

```diff
@@ -594,15 +602,22 @@ def test_circuit_with_aliased_cregs(self):
         qc.x(2).c_if(c2, 1)
         qc2 = QuantumCircuit(5, 5)
         qc2.compose(qc, [0, 2, 3], [2, 4], inplace=True)
-        cregs = [creg.name for creg in qc2.cregs]
+        qc3 = QuantumCircuit.copy_empty_like(qc2)
+        qc3.ry(np.pi / 4, 2)
+        qc3.cx(2, 1)
+        qc3.cx(0, 1)
+        qc3.h(0)
+        qc3.measure(0, 2)
+        qc3.measure(1, 4)
```

Comment:
it looks like qc2 is not used, so maybe `qc3` can replace `qc2` directly?

Reply:
If there is more straightforward way to make aliased registers, I would like to use it. This case does not look so intuitive.

Reply:
got it, I am not so familiar with aliased registers either, but the comment already makes it much clearer 👍